src/torrent.cpp

```diff
@@ -2775,40 +2825,51 @@ namespace libtorrent {
 			}
 #endif
 
-#ifndef TORRENT_DISABLE_LOGGING
-			debug_log("==> TRACKER REQUEST \"%s\" event: %s abort: %d"
-				, req.url.c_str()
-				, (req.event == tracker_request::stopped ? "stopped"
-					: req.event == tracker_request::started ? "started" : "")
-				, m_abort);
+				req.outgoing_socket = aep.socket;
```

Comment:
No it's not. I left it this way intentionally to keep the diff clean. I meant to create a separate commit to fix it but forgot. Thanks for reminding me.